py/selenium/webdriver/chromium/service.py

```diff
@@ -24,15 +24,16 @@ class ChromiumService(service.Service):
     """
 
     def __init__(self, executable_path, port=0, service_args=None,
-                 log_path=None, env=None, start_error_message=None):
+                 log_path=None, env=None, start_error_message=None, create_no_window=False):
```

Comment:
This would be better served as a method/property that is set when people don't want to a window.

Reply:
Thankyou for your response, I will try to make this a good fit in the Options class (for chrome)